### ISSUE TITLE
Add Docker enviroment to use during the development process

### ISSUE THREAD

**Issue by jairsjunior** [kafka-security/oauth#6]
Hi!

I think that add a Docker environment with (Zookeeper, Kafka and Keycloak) can help the developers during the tests with the new jar files created. For me looks to much faster to setup the environment (docker-compose up) and reduce the problems with some particular setups in each machine. 
What do you think about this?

I will be happy to contribute with this.

**Comment by shumphre** [comment]
Yes, I think an option to run in Docker would speed up local development for most developers.  I think we need to keep both options because some corporate developers use VDIs that don't not allow virtualization which prevents Docker from being able to run.

Have a way to run in Docker would be great!!
________________________________

**Comment by jairsjunior** [comment]
It's very nice think about this developers that cannot use virtualization. 
I will work on this docker setup. 

Do you have some concern about using the Confluent official opensource images?

**Comment by shumphre** [comment]
I'm not sure what you are referring to about the Confluent official open source images, can you give me some context?
________________________________

**Comment by jairsjunior** [comment]
Oh, sure!

[Confluent](https://www.confluent.io/about) is a company that was founded by the kafka creators and they have a apache 2.0 license images to use with docker. [You can see here](https://github.com/confluentinc/cp-all-in-one/tree/5.5.0-post/cp-all-in-one-community) and [here](https://github.com/confluentinc/kafka-images)

My doubt is there any problem for you to use this docker images, because it's developed by another company or something like this?

**Comment by jairsjunior** [comment]
What do you think about this PR? is this helpful?

**Comment by shumphre** [comment]
I have not had time to review yet.  I'm hoping to get to it this week.

Thanks
Steve
________________________________
